RevenueCat.podspec

```diff
@@ -24,6 +24,7 @@ Pod::Spec.new do |s|
   
   s.pod_target_xcconfig = {
     'DEFINES_MODULE' => 'YES',
+    'SWIFT_ACTIVE_COMPILATION_CONDITIONS' => 'ENABLE_VIRTUAL_CURRENCIES',
```

Comment:
We needed to enable the `ENABLE_VIRTUAL_CURRENCIES` compiler flag to access the `VirtualCurrencyInfo` struct. This will be removed when we eventually merge into main